server/services/reportedAdjudicationsService.ts

```diff
@@ -124,4 +133,48 @@ export default class ReportedAdjudicationsService {
       ),
     }
   }
+
+  async getPrisonerReport(
+    user: User,
+    adjudicationNumber: number,
+    locations: PrisonLocation[]
+  ): Promise<PrisonerReport> {
+    const newDraftAdjudicationData = await new ManageAdjudicationsClient(
```

Comment:
What is the correct behaviour here if we hit this url twice? Does createDraftFromCompleteAdjudication simply return the adjudication that had been recreated the first time do you know?

Reply:
Or do we need to check that one doesn't already exist?

Reply:
This is one of the things I wanted to check with you both. It just creates a new one every time - I thought about adding a check in there, but we have no information to check with? We don't have the new draft id that was created the first time we hit it...

Reply:
To quote my ticket for the work https://dsdmoj.atlassian.net/browse/NN-3780:

"ATM this endpoint will not make any effort to prevent duplicate drafts of the same reported adjudication".

It is the responsibility of the API to do the check, but it gets really complicated when you consider they might (somehow) have made changes to the existing draft.